tensorflow-base-cpu-linwin/def_bazelrc

```diff
@@ -1,3 +1,5 @@
+startup --output_user_root=C:/tmp
```

Comment:
I can add this as a comment as well but this it take from here: https://docs.bazel.build/versions/main/windows.html#avoid-long-path-issues